### DISCUSSION TITLE
下载超出限制被禁止

### DISCUSSION THREAD

**Discussion by AFewMoon** [the1812/Bilibili-Evolved#1376] [功能建议 / Ideas]
有时下载较大视频会被限制，请问能否解除限制？

如果前面有提及请指出链接。

版本：Stable v1.11.5

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved/blob/master/aria2-notice.md
#153，#248
![image](https://user-images.githubusercontent.com/49396421/103154209-e115bd00-47d0-11eb-98d5-24400dac3b3c.png)
